src/components/EscapeOutside/EscapeOutside.js

```diff
@@ -14,15 +14,15 @@ class EscapeOutside extends React.Component {
   }
 
   handleClick = e => {
-    const { onEscapeOutside } = this.props
-    if (!this.node.contains(e.target) && onEscapeOutside) {
+    const { onEscapeOutside = () => {} } = this.props
```

Comment:
A default prop is better than a default assignment, as it avoids recreating the `noop` on each method invocation.